### PR TITLE
Implement basic navigation bar with DOM updates

### PR DESCRIPTION
**Pull request by dominicrutk**

### CODE REVIEW

File: src/docs/milestone-01/styles.css [hunk 1]
```diff
@@ -1,12 +1,59 @@
...
+    button#navigation-toggle {
+      background: none;
+      border: none;
+      color: white;
+      cursor: pointer;
+      font-size: 16px;
+      padding: 4px 0;
+
+      &:hover {
+        color: darkgray;
```
Comment (angelikaladia) [inline src/docs/milestone-01/styles.css:32]: Should probably make all the hover the same color. I used this:  rgba(60, 62, 60, 0.249)

File: src/docs/milestone-01/styles.css [hunk 2]
```diff
@@ -1,12 +1,59 @@
...
+
+      li {
+        padding: 2px 0;
+
+        a {
+          color: white;
+          text-decoration: none;
+
+          &:hover {
+            color: darkgray;
```
Comment (angelikaladia) [inline src/docs/milestone-01/styles.css:49]: Same as above

**Review by angelikaladia — APPROVED** [review]

File: src/docs/milestone-01/styles.css (hunk 1 above)
Comment (dominicrutk) [reply src/docs/milestone-01/styles.css]: Switched to `#ccc` becuase both `darkgray` and `rgba(60, 62, 60, 0.249)` seem too dark on a green background.

File: src/docs/milestone-01/styles.css (hunk 2 above)
Comment (dominicrutk) [reply src/docs/milestone-01/styles.css]: Same as above

**Review by cjmitchell44 — APPROVED** [review]

**Pull request merged**